notebooks/210-slowfast-video-recognition/210-slowfast-video-recognition.ipynb

```diff
@@ -0,0 +1,567 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #38.        post_act = torch.nn.Softmax(dim=1)
</pre><p>I dont know can we use numpy instead of torch in this data process pipeline, so we can deploy openvino model without torch dependency</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/922/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
Replaced torch with numpy in the data process pipeline. 
The only use of torch.tensor is to feed input to the PyTorch model.

Additionally, CI tests are failing due to unrelated notebooks.